packages/react-app/src/components/pages/Bounties/Bounty/index.tsx

```diff
@@ -136,7 +139,7 @@ const BountyDetails = ({ bounty }: { bounty: BountyCollection }): JSX.Element =>
 			</GridItem>
 			<GridItem>
 				<Heading size="sm">Description</Heading>
-				<Text>{description}</Text>
+				<Text>{ ReactHtmlParser(DOMPurify.sanitize(toHTML(description))) }</Text>
 			</GridItem>
 			<GridItem>
 				<Heading size="sm">Done Criteria</Heading>
```

Comment:
Can you add it to the next line also?

Reply:
You mean to indent it like this:
```				
<Text>
    { ReactHtmlParser(DOMPurify.sanitize(toHTML(description))) }
</Text> 
```

Reply:
Do you mean I should add it to Done criteria?

Reply:
Yes, sorry. Github doesn't allow me to put comments on lines below this window.